internal/node/test/BUILD.bazel

```diff
@@ -604,3 +604,102 @@ nodejs_test(
         "@bazel_tools//src/conditions:windows": "@node16_windows_amd64//:node_toolchain",
     }),
 )
+
+# these rules create different targets for each toolchain that is supplied in the list for toolchains
+# this makes it easy to build or test against multiple versions of a toolchain at the same time with less duplicate work
+
+# this will create 2 nodejs_binary targets
+# bazel build //internal/node/test:main_multiple_loop_0
+# bazel build //internal/node/test:main_multiple_loop_1
+# the numbers are based on the order that the toolchains are passed to the rule
+[
+    nodejs_binary(
```

Comment:
duplicate targets to what is stamped out below?
```
//internal/node/test:main_multiple_loop_node16
//internal/node/test:main_multiple_loop_node15
```
?

Reply:
Yes they would be the same. Added to show different ways of doing the same thing. If there is a preferred way I can remove the other

Reply:
I think the `for id, tc in zip(` example with the clearer named targets is a better one so you can remove the `enumerate` one.